string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,14 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  for (var i = 0; i < word.length; i++) {
+    if (word[i].charCodeAt(0) <= 90 && word[i].charCodeAt(0) >= 65) {
```

Comment:
Same thing here, you should make use of `toUpperCase` and `toLowerCase` as needed.